### ISSUE TITLE
Can I generate the file flow.flo?

### ISSUE THREAD

**Issue by shanpingmao** [sniklaus/softmax-splatting#7]
Is it able to generate the file flow.flo for other test images? Thank you.

**Comment by sniklaus** [comment]
You can use any optical flow estimator that you find suitable for your task, for example the ones below.

https://github.com/sniklaus/pytorch-spynet
https://github.com/sniklaus/pytorch-pwc
https://github.com/sniklaus/pytorch-liteflownet
https://github.com/sniklaus/pytorch-unflow
